source/Armament.h

```diff
@@ -81,11 +96,31 @@ class Armament {
 
 
 private:
-	// Note: the Armament must be copied when an instance of a Ship is made, so
-	// it should not hold any pointers specific to one ship (including to
-	// elements of this Armament itself).
+	void RecreateViewsAndRanges();
+	// Return global minimum and maximum range of all weapons
+	// and the maximum range of turrets
+	std::tuple<double, double, double> CalculateRanges() const;
+
```

Comment:
```suggestion

private:
```

That's just a style thing we do.

Reply:
...and I missed :P I will do

Reply:
Done